schema.sql

```diff
@@ -13,5 +13,40 @@ CREATE TABLE animals (
 ALTER TABLE animals
 ADD species varchar(255);
 
+-- day3
+
+CREATE TABLE owners (
+    id SERIAL PRIMARY KEY,
+    full_name varchar(100) NOT NULL,
+    age INT NOT NULL
+);
+
+CREATE TABLE species (
+    id SERIAL PRIMARY KEY,
+    name varchar(100) NOT NULL
+);
+
+
+ALTER TABLE animals
```

Comment:
- I noticed that this query is not correct, if you wanted to remove the species column which is Varchar, not id , your query should look like this **ALTER TABLE tbl_name DROP COLUMN col_name;** . Please fix it in order to make query works as expected  and also add more query 👍